project/dexter-server/public/defect/defectCtrl.js

```diff
@@ -381,9 +381,13 @@ defectApp.controller('DefectCtrl', function ($scope, $http, $sce, $location, $an
         });
     }
 
-    $scope.toggleShowFileTreeBtn = function(isShowState) {
+    function toggleShowFileTreeBtn(isShowState) {
```

Comment:
What's the difference between befor and after?

Reply:
There is a bug in the 'defectCtrl.js' [▶ReferenceError: toggleShowFileTreeBtn is not defined]
Because I had changed function 'toggleShowFileTreeBtn' to '$scope.toggleShowFileTreeBtn'.
So I want to mapping it.